conf/modules.config

```
@@ -439,7 +448,28 @@ process {
     }
 
     withName: 'SUMMARIZE_RESULTS' {
-        ext.args   = "--columns sequence,score,score_type,psm,rt,mz,charge,accessions,aa_before,aa_after,start,end,COMET:deltaCn,COMET:deltaLCn,COMET:lnExpect,COMET:xcorr,rt_diff_best,observed_retention_time_best,predicted_retention_time_best,spec_pearson,std_abs_diff,ccs_observed_im2deep,ccs_predicted_im2deep,ccs_error_im2deep"
+        ext.args   = "--columns peptidoform,sequence,score,score_type,psm,rt,mz,charge,accessions,aa_before,aa_after,start,end,COMET:deltaCn,COMET:deltaLCn,COMET:lnExpect,COMET:xcorr,rt_diff_best,observed_retention_time_best,predicted_retention_time_best,spec_pearson,std_abs_diff,ccs_observed_im2deep,ccs_predicted_im2deep,ccs_error_im2deep"
```

```suggestion
        ext.args   = [
            "--columns ",
            [
                "peptidoform",
                "sequence",
                "score", "score_type",
                "psm",
                "rt",
                "mz",
                "charge",
                "accessions",
                "aa_before", "aa_after",
                "start", "end",
                "COMET:deltaCn", "COMET:deltaLCn", "COMET:lnExpect", "COMET:xcorr",
                "rt_diff_best",
                "observed_retention_time_best", "predicted_retention_time_best",
                "spec_pearson",
                "std_abs_diff",
                "ccs_observed_im2deep", "ccs_predicted_im2deep", "ccs_error_im2deep"
            ].join(',').trim(),
        ].join(' ').trim()
```
Putting and grouping colums on separate lines for better readability